src/main/java/bhs/devilbotz/commands/auto/MobilityDockAndEngage.java

```diff
@@ -56,7 +61,8 @@ public MobilityDockAndEngage(
 
     addCommands(CommandDebug.start());
     addCommands(drivetrain.followTrajectoryCommand(path, true, false));
-    addCommands(new DockAndEngage(drivetrain, 2));
+    // Since the trajectory causes the robot to turn 180 degrees, we set the targetAngle accordingly
+    addCommands(new DockAndEngage(drivetrain, 2, 180 - startAngle));
```

Comment:
I'm not great at math... this does not seem intuitively right.  If robot started at 90.. then opposite would be 270? This line would make it still be 90? Can you explain why `180-startAngle` works, in a comment in the code please?

Reply:
The `startAngle` is the yaw/angle that the robot was in when the autonomous command is started (i.e. at the beginning of the match).  Since Mobility drives in a curved path, the DockAndEngage should be exactly opposite of the starting orientation.

![image](https://user-images.githubusercontent.com/49494444/225413103-e35e2c3a-2b85-4ed4-8977-3cc10d731bd6.png)

Reply:
It's fixed in the latest code.  It just happened to work "as-is" because we were starting at either "0" or "180".  The "90" degree example exposed the obvious logic error.  Tnx!